### ISSUE TITLE
Logging computer lock/unlock events

### ISSUE THREAD

**Issue by irhs** [gurgeh/selfspy#121]
It would be interesting to also log

- display on/off events,
- screensaver start/stop events, and
- computer lock/unlock events.

This info can also be used to give a more accurate estimate for pactive. The current approach uses 2 min from click/key timings, which gives a wrong estimates when processes used for watching a video or reading. If we know the times when computer display was off or computer was locked, then we might be able to give a more accurate estimate for `pactive`

**Comment by gurgeh** [comment]
Yes, this would be interesting. Unfortunately it would mean even more hairy, difficult to test, platform specific code. For Linux it would probably also mean different approaches for different window managers (though maybe it is doable from proc or kernel logs or something).

**Comment by lexBuright** [comment]
`xscreensaver` (and probably `gnome-screensaver` - since it seems to mostly be `xscreensaver` with a different name) has a built-in way of doing this:

``` sh
xscreensaver-command  -watch
LOCK Thu Aug 18 14:23:02 2016
RUN 159
UNBLANK Thu Aug 18 15:23:44 2016
^C
```

This means the approach probably wouldn't be that hairy, fragile or difficult to test (assuming you trust `xscreensaver` :) ). It would indeed be platform specific and more code though. 

What's your take on features that are only implemented for one platform?
